lib/ansible/modules/windows/win_file.py

```diff
@@ -28,32 +28,53 @@
 ---
 module: win_file
 version_added: "1.9.2"
-short_description: Creates, touches or removes files or directories.
+short_description: Sets attributes of files
 description:
-     - Creates (empty) files, updates file modification stamps of existing files,
-       and can create or remove directories.
-     - Unlike M(file), does not modify ownership, permissions or manipulate links.
+  - Sets attributes of files, symlinks, hard links, junction points and directory.
+  - Can create files, symlinks, hard links, junction points and directories.
+  - Unlike M(file), does not modify ownership, permissions or manipulate links.
```

Comment:
I'll change this tomorrow when I've got access to a computer, thanks for the pickup.